### ISSUE TITLE
Not working in Tensor Flow

### ISSUE THREAD

**Issue by shekit** [llSourcell/linear_regression_live#4]
Hi,

I tried implementing this dataset using tensor flow using the linear regression example provided in the get started docs of tensor flow (not the one using tf.contrib.learn) . While the provided example works, if I use any other data as an input to train on I always get the following printed out:

`W: [ nan] b: [ nan] loss: nan`

I have tried it with various different data sets. I even reduced the dataset Siraj provided to just the first five elements in integer form

 ```
#training data
  x_train = [32,53,61,47,59]
  y_train = [31,68,62,71,87]
```

I can implement linear regression on the data without any problem if I implement it only in numpy and get correct weight and bias values.

I have tried adjusting the hyperparameters but still no luck. It always returns nan

I have also literally copied the code from the tensorflow site and just replaced the data values so I know there is no hidden typo. This has been driving me crazy. Can someone please try this?

**Comment by shekit** [comment]
Turns out the learning rate needed to be lowered to .000001 before I got any results. Though when implemented in numpy the learning rate of just .01 was sufficient to get results. Any ideas why?

**Comment by shekit** [comment]
Normalizing the data in a preprocessing step will help avoid this
